src/schema/items/recording.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name: Recording
+class: entity
+description: |
+  More than one continuous recording file can be included (with different
+  sampling frequencies).
+  In such case use different labels.
+  For example: `_recording-contrast`, `_recording-saturation`.
```

Comment:
unrelated to this PR, I wonder if we should just reserve `examples:` key in the records to contain a list of possible examples?  That would de-mix description from examples, and allow to harmonize rendering (ATM it could be "For example:", "e.g.", etc)

Reply:
I'd like to avoid incorporating examples into the schema files since a lot of the examples used throughout the specification for different entities is section-specific. For example, `acq` has different examples for anatomical MRI and functional MRI in the text.

I think you're right, though, that the examples should be removed from the entity definitions. We should clean up those definitions.

Reply:
I wrote a response nodding in agreement but then realized -- we are talking about the super dooper machine readable filenaming standard!   If we have ANY examples we would anyways need to parse and validate them to be legit or specify them as records (`{'datatype': 'anat', 'subject': '01', 'acq': 'blah', 'suffix: 'T1w', 'extension': '.nii.gz'}`) to start with and render when presented (as `sub-{subject}/{datatype}/...`) since we know entities order etc.  Then any section of the bids specification "manual" could just limit examples to a specific filter (`datatype: anat`), and provide only matching examples. But then I also see a benefit from documentation providing examples across different datatypes for a given entity in data type agnostic section ;)

but anyways -- it is  indeed a separate topic for some future work and indeed they just need to be removed from descriptions.

Reply:
The filename format rendering will be generated in the specification text in #610. The examples I'm referring to are more text-based, like "acq may refer to something like different resolutions of T1w scans" in the **Anatomical MRI** section, while in the MEG `_photos` section it might say that acq is good for distinguishing "acquisition of different photos of the same face". Those kinds of examples are highly section-specific and (IMHO) shouldn't be in the schema.